### DISCUSSION TITLE
Is there a way to start shizuku with dhizuku?

### DISCUSSION THREAD

**Discussion by HallowedSpace** [iamr0s/Dhizuku#78] [Q&A]
Is there a way to start shizuku with dhizuku? Asking because dhizuku alone isnt supported as much as shizuku, and id like to see dhizuku be able to start shizuku autonomously.

**Comment by youfeng11** [comment]
The DeviceOwner permission is insufficient to start Shizuku.

**Reply by HallowedSpace** [reply]
Why so?

**Reply by HallowedSpace** [reply]
Is it possible to open an ADB shell with Dhizuku?

**Reply by j-romchain** [reply]
No, Dhizuku is device owner not adb. Device owner is an admin account on the phone, and is how things like parental controls are configured, and has some more but far from all permissions, it can also be given to an app and as such does not need to be reran every reboot. ADB is meant for developers debugging apps or Android itself, and as such has access to even more and stranger permissions, but in order to access it you either need to run the debug bridge, or hack into its permission level, which isn't even hacking from root, but you would need to actually hack in order to persist between reboots. Root is true admin with full control, but on android you either need a specially compiled version or a modified version.

**Comment by GENM796** [comment]
Is it still not possible in version 2.11.2 Or in the lower version??

**Reply by youfeng11** [reply]
This is impossible, no matter what.
